src/components/side_panel/chart/pyramid_chart/pyramid_chart_config_panel.ts

```diff
@@ -0,0 +1,9 @@
+import { BarConfigPanel } from "../bar_chart/bar_chart_config_panel";
+
+export class PyramidConfigPanel extends BarConfigPanel {
```

Comment:
You should probably extend the `GenericChartConfigPanel` instead, the only think the bar panel adds is the handling of the "stacked" checkbox (which shouldn't be here)